doc/js_libs.pl.md

```diff
@@ -0,0 +1,183 @@
+# Biblioteki JS
+
```

Comment:
Can you add some description to general way how all plgapp js libraries are created? E.g. you can mention about this `errback` convention here.